api/hypershift/v1beta1/hostedcluster_types.go

```diff
@@ -1823,6 +1833,12 @@ type AzurePlatformSpec struct {
 	SecurityGroupID string `json:"securityGroupID,omitempty"`
 }
 
+// OpenStackPlatformSpec specifies configuration for clusters running on OpenStack.
+type OpenStackPlatformSpec struct {
```

Comment:
Please don't review the spec here. We changed it later and this will be proposed in a new PR after this one.
Thanks

Reply:
@EmilienM Just a warning: although we plan to change it and we don't expect anybody to actually use it...

We should still only commit backwards-compatible changes to Hypershift unless there are exceptional circumstances, as it can still cause grief.

Are these 2 fields in their expected final form? If not, lets remove them.

Reply:
I'll be careful the next time; however I suggest we move forward here. I really don't want to burn unnecessary CI jobs here to iterate again on this one.
The next PR is going to update that.